dangerfile.ts

```diff
@@ -0,0 +1,23 @@
+// https://danger.systems/js/reference.html
+import type {PackageShape} from './.danger/types';
+import packageJsonCheck from './.danger/package-json-check';
+import missingChangesetsCheck from './.danger/missing-changesets-check';
+import changesetsThatNeedCoreCheck from './.danger/changesets-that-need-core-check';
+
+// because it's not typescript
```

Comment:
Could I have more context about this comment?

Reply:
Basically this is a typescript file, and our typescript config is quite strict. The old mono repo tools were written in untranspiled JavaScript, and when you try to import them as a module in Typescript, it basically doesn't like them. So you have to require them as commonJS modules.

Reply:
ah, so this is the same as the "typsecript doesn't like JS imports" comments I've seen sometimes, is that right?